test/assets/preset-mini-targets.ts

```diff
@@ -848,13 +848,18 @@ export const presetMiniTargets: string[] = [
   'transition-colors,opacity-200',
   'transition-color,background-color-200',
   'transition-background-color,color-200',
+  'transition-$variant',
+  'transition-[width_height]',
+  'transition-[width_height_colors]',
+  'transition-[width_height_colors]-200',
   'transition',
   'transition-revert-layer',
   'property-none',
   'property-all',
   'property-unset',
   'property-margin,padding',
   'property-padding,margin',
```

Comment:
`property-padding,margin`
 This will look a bit fragmented, and there may be some unknown regular expression parsing problems. It is recommended not to support it and keep `property-[padding, margin]`

Reply:
Uno users have been writing `transition` rules in this style for a long time, so I have adopted a compatible way of writing. Otherwise, it will be a breaking change.

Reply:
I looked at the previous tests, well, I haven't seen it used like this before, I thought it was a feature